core/layout_api.php

```diff
@@ -655,19 +647,12 @@ function layout_navbar_subproject_option_list( $p_parent_id, $p_project_id = nul
 			$t_full_id = $t_id;
 		}
 
-		echo '<li>';
+		echo 0 == strcmp( $p_project_id, $t_full_id ) ? '<li class="active">' : '<li>';
```

Comment:
Not needed here since $p_project_id & $t_full_id are strings - but I wanted to keep it consistent.